frontend/src/services/questionApi.ts

```diff
@@ -0,0 +1,46 @@
+import { rootApi } from "@/app/RootApi.ts";
+
+import type { QuestionType } from "@/features/questions";
+
+rootApi.enhanceEndpoints({ addTagTypes: ["Question"] });
+
+const buildServiceUrl = (queryUrl: string) =>
+  `${process.env.NEXT_PUBLIC_QUESTION_SERVICE_ADDRESS}${queryUrl}`;
+
+const questionApi = rootApi.injectEndpoints({
+  endpoints: (builder) => ({
+    getQuestions: builder.query<QuestionType[], Partial<QuestionType> | void>({
```

Comment:
I realized there is a better way to handle this recently! I will update you on it.